pxr/usd/usdLightField/schema.usda

```diff
@@ -0,0 +1,440 @@
+#usda 1.0
+(
+    subLayers = [
+        @usd/schema.usda@,
+        @usdGeom/schema.usda@
+    ]
+)
+
+over "GLOBAL" (
+    customData = {
+        string libraryName       = "usdLightField"
```

Comment:
Late to the particle field vs light field conversation; I think particle field makes more sense personally.  But should the directory be usdParticleField as well?